src/Microsoft.AspNet.Hosting/Internal/AutoRequestServicesStartupFilter.cs

```diff
@@ -14,7 +14,10 @@ public Action<IApplicationBuilder> Configure(IApplicationBuilder app, Action<IAp
             return builder =>
             {
                 app.UseMiddleware<RequestServicesContainerMiddleware>();
-                next(builder);
+                if (next != null)
```

Comment:
Probably was an edge case with start up filters/mangled app pipelines, but I hit this yes...